Code/GraphMol/ForceFieldHelpers/MMFF/MMFF.h

```diff
@@ -93,7 +93,7 @@ void MMFFOptimizeMoleculeConfs(ROMol &mol,
     delete ff;
   } else {
     for (unsigned int i = 0; i < mol.getNumConformers(); ++i) {
-      res[i] = std::make_pair(static_cast<int>(-1), static_cast<double>(-1));
+      res.emplace_back(std::make_pair(static_cast<int>(-1), static_cast<double>(-1)));
```

Comment:
This is going to do something ugly if `res` already has elements in it.
I'd suggest doing `res.resize(mol.getNumConformers())` on line 95 and then leaving this line as is.
Or do `res.resize(0)` on line 95. Either way.

Reply:
Done;indeed that's how `OptimizeMoleculeConfs` already works.